spec/models/medicaid_application_member_attributes_spec.rb

```diff
@@ -110,11 +121,43 @@
         ).to_h
 
         expect(result.keys).not_to include(
-          %i[
-            primary_member_birthday
-            primary_member_under_21_no
-            primary_member_under_21_yes
-          ],
```

Comment:
This was asserting that the array of keys was not one item contained in `.keys`. In other words it would never have caught the keys.